src/Build/Evaluation/Evaluator.cs

```diff
@@ -652,7 +652,7 @@ private void Evaluate()
                 List<string> initialTargets = new List<string>(_initialTargetsList.Count);
                 foreach (var initialTarget in _initialTargetsList)
                 {
-                    initialTargets.Add(EscapingUtilities.UnescapeAll(initialTarget.Trim()));
+                    initialTargets.Add(EscapingUtilities.UnescapeAll(initialTarget, true));
```

Comment:
nit: I prefer to include the parameter name for bool literal arguments to make it clear what the bool means without having to look at the signature of the called method.